static/js/featuredialog.js

```diff
@@ -0,0 +1,45 @@
+var FeatureDialog = Backbone.View.extend({
+	className: "marker-dialog",
+	events: {
+		"submit" : "submit",
+		"click .save-button" : "submit",
+		"click .close-button" : "close"
+	},
+	initialize: function(options) {
+		this.markers = options.markers;
+		this.type = options.type;
+		this.event = options.event;
+
+	},
+	render: function() {
+		this.$el.append($("#feature-dialog-template").html());
+		$(document.body).append(this.$el);
+		this.modal = this.$el.find(".modal");
+		this.modal.modal("show");
+		return this;
+	},
+	close: function() {
+		this.modal.modal("hide");
+	},
+	submit : function()
+    {
+        var adr = this.$el.find("#emailAddress").val();
+        var firstname = this.$el.find("#fName").val();
+        var lastname = this.$el.find("#lName").val();
+        $.ajax({
```

Comment:
Doesn't work with post - I think because contentType:"application/json" is required